File: backend/src/main/java/com/hotelbooking/backend/service/FacilityService.java
// File: src/main/java/com/hotelbooking/backend/service/FacilityService.java
package com.hotelbooking.backend.service;

import com.hotelbooking.backend.dto.facility.FacilityDto;
import com.hotelbooking.backend.model.Facility;
import java.util.List;

public interface FacilityService {
    Facility addFacility(FacilityDto facilityDto);
    List<Facility> getAllFacilities();
}

File: frontend/frontend/src/services/facilityService.js
// src/services/facilityService.js
import api from './api';

export const getAllFacilities = () => {
    return api.get("/api/public/facilities");
};

// Add this new function
export const addFacility = (facilityData) => {
    return api.post("/api/admin/facilities", facilityData);
};